test/integration/targets/nxos_igmp_interface/tests/common/sanity.yaml

```diff
@@ -46,9 +46,9 @@
       report_llg: true
       immediate_leave: true
       group_timeout: 300
-      # deprecated
-      oif_prefix: 239.255.255.2
-      oif_source: 192.0.2.1
+      oif_ps:
```

Comment:
The logic is a bit more complex for `oif_ps` since you can have any of the following.

1. Source and prefix.
2. Prefix with no source.
3. List of source/prefix dictionaries or list of prefix dicts with no source or combination.

I would like to see an integration test that covers all 3 scenarios above.

Reply:
Some of that is already here: https://github.com/ansible/ansible/blob/8020fef8391617bc16fc94f7e372ebf6bc6c976a/test/integration/targets/nxos_igmp_interface/tests/common/sanity.yaml#L67-L85

I don't really see how that is related to this PR though. This PR only removes two options which have been deprecated for a long time. I think adding more tests for the module makes more sense in another PR.

Reply:
> Some of that is already here:
> 
> https://github.com/ansible/ansible/blob/8020fef8391617bc16fc94f7e372ebf6bc6c976a/test/integration/targets/nxos_igmp_interface/tests/common/sanity.yaml#L67-L85
> 
> I don't really see how that is related to this PR though. This PR only removes two options which have been deprecated for a long time. I think adding more tests for the module makes more sense in another PR.

I missed the additional tests and thanks for pointing them out.  I am happy with this coverage.  While I agree in principle that the scope of adding tests may be a bit beyond this PR of removing deprecated parameters, there is a chance the the old parameters might have been used by old playbooks instead of converting to the single parameter.  Just wanted to make sure we had good coverage for the single param.